modules/modules.sqf

```diff
@@ -46,6 +46,7 @@ Additional modules that can be enabled by removing the //
 //#include "gas\root.sqf"
 //#include "grad-fortifications\root.sqf"
 #include "headless_ai\root.sqf"
```

Comment:
```suggestion
//#include "headless_ai\root.sqf"
```

Reply:
You reckon headless AI should no longer be a default modul?

Reply:
yeah, loads things when not at all needed in TVT